docs/src/main/asciidoc/attributes.adoc

```diff
@@ -1,2 +1,3 @@
 :idprefix:
 :idseparator: -
+:icons: font
```

Comment:
Tested the preview on with the IntellIJ plugin, the `NOTE`, `WARNING` and `TIPS` blocks rendered correctly.